src/server.c

```diff
@@ -3398,6 +3398,30 @@ void call(client *c, int flags) {
     server.stat_numcommands++;
 }
 
+void rejectCommand(client *c, robj *reply) {
```

Comment:
This functions are non obvious in what they are useful for, yet they lack a top comment.